admin/data/sql/migrations/074.do.create-auditlog-table.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE mtc_admin.auditLog
+(
+	id int identity primary key,
```

Comment:
Any idea if an `int` will be large enough to store all audit's or should we move to `bigint` now?

Reply:
good point - definitely should be `bigint`